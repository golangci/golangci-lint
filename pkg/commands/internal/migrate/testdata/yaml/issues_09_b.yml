issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

  new: false
  new-from-merge-base: ''
  new-from-rev: ''
  new-from-patch: ''
  whole-files: false
  fix: false
