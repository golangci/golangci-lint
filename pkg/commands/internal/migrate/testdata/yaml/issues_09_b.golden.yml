version: "2"
issues:
  new-from-rev: ""
  new-from-merge-base: ""
  new-from-patch: ""
  whole-files: false
  new: false
  fix: false
